helm/starboard-app/values.yaml

```
@@ -34,13 +42,13 @@ starboard-app:
         memory: 1G
 
   kubeBench:
-    imageRef: quay.io/giantswarm/kube-bench:0.6.3
+    imageRef: docker.io/giantswarm/kube-bench:0.6.3
 
   polaris:
-    imageRef: quay.io/giantswarm/polaris:4.0
+    imageRef: docker.io/giantswarm/polaris:4.0
 
   conftest:
-    imageRef: giantswarm/conftest:v0.25.0
+    imageRef: docker.io/giantswarm/conftest:v0.25.0
 
   aqua:
     imageRef: giantswarm/scanner:5.3
```

```suggestion
    imageRef: docker.io/giantswarm/scanner:5.3
```

Can we have the same registry structure to keep it consistent?